lib/auth/tls_test.go

```diff
@@ -1080,7 +1080,7 @@ func TestAppServerCRUD(t *testing.T) {
 	ctx := context.Background()
 	tt := setupAuthContext(ctx, t)
 
-	clt, err := tt.server.NewClient(TestBuiltin(types.RoleApp))
+	clt, err := tt.server.NewClient(TestBuiltin(types.RoleAdmin))
```

Comment:
The same as above. Why updating the role version require to elevate the privilege here?

Reply:
I changed this test before giving `RoleApp` explicit wildcard allow `AppLabels`, which was a mistake. I've now reverted the change in this test.